nucypher/characters/base.py

```diff
@@ -74,7 +76,11 @@ def __init__(self,
 
         """
 
-        Base class for Nucypher protocol actors.
+        A participant in the cryptological drama (a screenplay, if you like) of NuCypher.
+
+        Characters can represent users, nodes, wallets, offline devices, or other objects of varying levels of abstraction.
```

Comment:
I... am incredibly opposed to this line. :joy:

Reply:
I don't know, I just don't agree that a character is accurately described in this line lol.

Reply:
Eh?  What's inaccurate here?